ocrd_utils/ocrd_logging.conf

```diff
@@ -52,6 +53,19 @@ handlers=consoleHandler
 #handlers=fileHandler
 #qualname=ocrd.workspace
 
+
```

Comment:
The PIL restriction from the builtin config omitted because it is `INFO` anyway, correct?

Reply:
Yes, since it is the default level. But If you mind, I can add this as well

Reply:
Shall `ocrd` be integrated, too? 
Be aware, this might have side-effects, because it is the far most top ocrd logging channel.

Reply:
> Shall `ocrd` be integrated, too?
> Be aware, this might have side-effects, because it is the far most top ocrd logging channel.

Since we don't specify logging behavior for `ocrd*` in the builtin config, we should not in the config file.

I also would leave the `[logger_ocrd_workspace]` commented out as an example - setting this to `DEBUG` by default will be very verbose.

Reply:
This makes sense, I adopt your suggestions asap